targets/XPlatCppSdk/source/cppsdk/include/playfab/PlayFabHttp.h

```diff
@@ -41,12 +44,17 @@ namespace PlayFab
         static void MakeInstance();
         ~PlayFabHttp() override;
```

Comment:
>PlayFabHttp [](start = 9, length = 11)

Is this susceptible to object slicing problem ? Can you double check ?

Reply:
It is not, in this particular case. Having them both virtual (in base class and descendant) eliminates this problem.

---
In reply to: [220350785](https://github.com/PlayFab/SDKGenerator/pull/295#discussion_r220350785) [](ancestors = 220350785)